### PR TITLE
Adding Instrumentation Key as a variable

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Adding Instrumentation Key as a variable

### JIRA link ###
 https://tools.hmcts.net/jira/browse/RPE-879 (Though directly not associated) 

### Change description ###

Added Instrumentation Key for App Insights . 
Variable is mapped to APPINSIGHTS_INSTRUMENTATIONKEY as per [applicationInsights package](https://www.npmjs.com/package/applicationinsights) so that apps can just call .setup().

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: nodejs/templates/deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ spec:
         name: {{ template "hmcts.releaseName" . }}
         {{- if .Values.environment }}
```
Comment (timja) [inline nodejs/templates/deployment.yaml]: this if statement will need moving right?

Comment (adusumillipraveen) [reply nodejs/templates/deployment.yaml]: it doesn't make sense after this change, but why before, the current one looks fine?

Comment (timja) [reply nodejs/templates/deployment.yaml]: true I would expect it to be a loop of 0 values and do nothing, but if its nil maybe it fails? so might still need that if check for the environment section?

Comment (timja) [reply nodejs/templates/deployment.yaml]: can you test what happens with no environment values? and if its fine we can merge otherwise needs a change

Comment (adusumillipraveen) [reply nodejs/templates/deployment.yaml]: I tested it 1) without having environment  2) having environment: but no values - making it environment: null in computed values . It works in both cases. Happy to add if we think it makes it more readable.

Comment (timja) [reply nodejs/templates/deployment.yaml]: nah is fine thanks

**Pull request merged**